### ISSUE TITLE
Socom 3: Video subtitles show all over the screen, + crash in Stockpile mission

### ISSUE THREAD

**Issue by hrydgard** [hrydgard/ppsspp#18094]
It's been previously reported but I can't find the issue, maybe it was only reported on discord. Was reported again now by ceda5.

It's been confirmed that if subtitles is on in this game, we end up with not only subtitles all over the screen, but a crash later in the Stockpile mission.

This should display as a single line on the bottom of the screen:

![image](https://github.com/hrydgard/ppsspp/assets/130929/4cb4707e-dbcc-4f80-a97e-a59892fb73b1)

ceda5 also showed a video of it happening in Tactical Strike .

https://github.com/hrydgard/ppsspp/assets/130929/347c37aa-fc82-4a27-994f-d7fdd1a375b3

**Comment by hrydgard** [comment]
Here's a GE frame dump showing the problem:

[UCES01242_0001.zip](https://github.com/hrydgard/ppsspp/files/13788958/UCES01242_0001.zip)

Unfortunately I don't really understand why it's like that. It really looks like it's intentionally drawing all the lines, and it's not setting up a scissor rectangle or something...

**Comment by sum2012** [comment]
Maybe PsmfPlayer bug
v1.16.6-708-ga416d94e3-windows-amd64 log
https://gist.githubusercontent.com/sum2012/d2300ffc50cc85cbe5c6f4f84ef0293e/raw/3e619289d0bac594266c817b89ebf76ac615adef/gistfile1.txt

**Comment by sum2012** [comment]
NOTICE_LOG(ME, "Total Steam %i", psmfplayer->totalVideoStreams);

Total steam is 1 , so may not be PsmfPlayer bug

**Comment by hrydgard** [comment]
Yeah, I would have guessed some bug reading subtitles from a 2nd stream, but likely not then. Really not sure how this can possibly happen then... would have to be a very strange CPU emulation bug..

**Comment by sum2012** [comment]
Even I set to "Interpreter" , still fail.
A note: JPCSP emulator no problem

**Comment by sum2012** [comment]
So that this game need "sceKernelSemaphore" support in scempeg ?

**Comment by hrydgard** [comment]
Well we have semaphore support, but some games wait for some semaphore that doesn't exist, and I'm not sure where the game tries to read the handle from.

It could also be something completely different, but it's almost certainly PSMF video player related somehow.

**Comment by nassau-tk** [comment]
Probably, Relation with #12730.

**Comment by hrydgard** [comment]
Please test according to this: #20200

**Comment by Saramagrean** [comment]
Can confirmed, game isn't crash after turn on "Disable HLE for scePsmf / scePsmfPlayer".

![Image](https://github.com/user-attachments/assets/9c0b0d46-6b54-490c-aec7-011320c24bcf)

**Comment by hrydgard** [comment]
And subtitles work normally in videos, if enabled?

**Comment by Saramagrean** [comment]
> And subtitles work normally in videos, if enabled? [@Saramagrean](https://github.com/Saramagrean)

Yeah, subtitles work fine. (display as a single line on the bottom of the screen)

**Comment by hrydgard** [comment]
Fixed since #20208

**Comment by FATCatAndroid12** [comment]
```
[DisableFirstFrameReadback]
# SOCOM: Fireteam Bravo 3 (see issue #18958, hopefully fixes crash in Stockpile mission, #10461)
UCES-01242 = true
NPJG-00035 = true
NPHG-00032 = true
```

This should be removed now since this game is finally fixed!

this is also belong to v1.19.0 milestone
